src/metadata/routes/schemas/template.clj

```diff
@@ -79,7 +79,11 @@
    (describe [TemplateAttrEnumValue] "The list of possible values for enumeration types")
 
    (s/optional-key :settings)
-   (describe TemplateAttrSettings "Arbitrary settings saved by an admin and used by the UI (e.g. OLS query parameters)")})
+   (describe TemplateAttrSettings "Arbitrary settings saved by an admin and used by the UI (e.g. OLS query parameters)")
+
+   (s/optional-key :attributes)
+   (describe [(s/recursive #'MetadataTemplateAttr)]
+             "Nested attributes (used for grouping related template attributes)")})
```

Comment:
It should also be possible to have an attribute with a value to have nested attributes, so I don't think it's strictly for grouping related attributes.

For example, the DataCite `title` attribute has a value, but it can also have a `titleType` attribute. So in our template, `titleType` would be a nested attribute of the `title` attribute. Another example is `relatedIdentifier` and `relatedIdentifierType`.

So nested attributes can also be attributes of a parent attribute.

Reply:
I struggled a bit with this one. I knew that cases like that existed, but trying to describe them in addition to grouping seemed a little much for swagger docs. I ended up shortening the documentation to say simply "Nested attributes." The minimalist approach is probably better here. 😄